go.mod

```diff
@@ -1,29 +1,29 @@
 module connectrpc.com/vanguard
 
-go 1.21
+go 1.22
+
+toolchain go1.23.4
 
 require (
 	buf.build/gen/go/connectrpc/eliza/connectrpc/go v1.11.1-20230822171018-8b8b971d6fde.1
 	buf.build/gen/go/connectrpc/eliza/grpc/go v1.3.0-20230822171018-8b8b971d6fde.1
 	buf.build/gen/go/connectrpc/eliza/protocolbuffers/go v1.31.0-20230822171018-8b8b971d6fde.1
-	connectrpc.com/connect v1.16.2
+	connectrpc.com/connect v1.18.1
```

Comment:
We should update `buf.gen.yaml` to use the same versions of plugins as the updated dependencies.

Reply:
Just updated.